pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator_test.go

```diff
@@ -1421,7 +1421,7 @@ var _ = Describe("VirtualMachineInstance Mutator", func() {
 			})
 
 			DescribeTable("should leave MaxGuest empty when memory hotplug is incompatible", func(vmiSetup func(*v1.VirtualMachineInstanceSpec)) {
```

Comment:
I think this DescribeTable can be modified so that it accepts a set of libvmi.Option that the entries will provide